### DISCUSSION TITLE
VSC instead of PLATFORMIO

### DISCUSSION THREAD

**Discussion by soky** [Mair/esp32-course#99] [Course Qeustions] [answered]
Hi, just curios as to why you chose VSC instead of PLATFORMIO for this Course.  Is VSC the easiest way ESP32?

Thank you.

soky157

**Comment by Mair** [comment]
Hi @soky,

Platform.io makes certain assumptions about how the chip is configured. It therefore does not allow you (or makes it harder) to have the level of control that using the IDF does. It does boil down to a matter of preference. I much prefer having less automated and more control. and the IDF extension does a reasonable job

**Comment by soky** [comment, marked as answer]
Thank you Mair very much for taking the time to answer my question.

soky
